.github/workflows/ios_ci.yml

```
@@ -0,0 +1,72 @@
+name: Fastlane CI
+
+on:
+  pull_request:
+    branches:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)